### PR TITLE
PolicyActionsRegistry

### PR DESCRIPTION
**Pull request by s2quake**

### CODE REVIEW

**Comment by greymistcube** [comment]
Changed the name `IsBlockAction` to `IsPolicyAction`. Having `BeginTxActions` and `EndTxActions` as "block action"s seemed non-sensical at this point. 🙄

File: src/Libplanet.Action/ActionEvaluator.cs [hunk 1]
```diff
@@ -243,6 +264,7 @@ public class ActionEvaluator : IActionEvaluator
             IActionContext context,
             IAction action,
             IStateStore stateStore,
+            bool isBlockAction,
```
Comment (limebell) [inline src/Libplanet.Action/ActionEvaluator.cs:267]:
```suggestion
            bool isPolicyAction,
```

File: src/Libplanet.Action/ActionEvaluator.cs [hunk 2]
```diff
@@ -265,10 +287,11 @@ IActionContext CreateActionContext(IWorld newPrevState)
                     miner: inputContext.Miner,
                     blockIndex: inputContext.BlockIndex,
                     blockProtocolVersion: inputContext.BlockProtocolVersion,
-                    txs: inputContext.Txs,
                     previousState: newPrevState,
                     randomSeed: inputContext.RandomSeed,
+                    isPolicyAction: isBlockAction,
```
Comment (limebell) [inline src/Libplanet.Action/ActionEvaluator.cs:292]:
```suggestion
                    isPolicyAction: isPolicyAction,
```

**Review by limebell — CHANGES_REQUESTED** [review]

File: src/Libplanet.Action/ActionEvaluator.cs (hunk 1 above)
Comment (s2quake) [reply src/Libplanet.Action/ActionEvaluator.cs]: amended https://github.com/planetarium/libplanet/pull/3882/commits/84682b3a0f41697f4986c58a7e92ed86c6af30a1

File: src/Libplanet.Action/ActionEvaluator.cs (hunk 2 above)
Comment (s2quake) [reply src/Libplanet.Action/ActionEvaluator.cs]: amended https://github.com/planetarium/libplanet/pull/3882/commits/84682b3a0f41697f4986c58a7e92ed86c6af30a1

**Review by limebell — APPROVED** [review]

**Pull request merged**